### PR TITLE
🐭 ADD - node pinning for gitlab 🐭

### PR DESCRIPTION
**Pull request by springdo**
As per the description, just a small rework to allow node pinning for gitlab. I checked in the spec for the machineset for somewhere to keep it

### CODE REVIEW

**Comment by eformat** [comment]
everyone happy to mege this ? @springdo @jfilipcz @pabrahamsson 

LGTM .. so will merge if no objections ...

File: tooling/charts/tl500-base/values.yaml [hunk 1]
```diff
@@ -138,3 +138,14 @@ gitops-operator:
 
 tl500-teamsters:
   enabled: true
+
+# assumes node-role.kubernetes.io/infra: "" on one or more nodes
+runOnInfra: false
+
+# not sure if this should be in this repo but putting somewhere for safe keeping at least
```
Comment (jfilipcz) [inline tooling/charts/tl500-base/values.yaml:145]: I would move that into README section with a note that's there only for AWS

**Review by jfilipcz — APPROVED** [review]
Perhaps updating the values.yaml file as per comment would be a good idea, but otherwise is LGTM

**Comment by springdo** [comment]
@jfilipcz / @eformat  bump. added the line in README.md

**Comment by ckavili** [comment]
are we good with this @jfilipcz @eformat ☺️☺️ (🧹🧹🧹)

**Pull request merged**